apps/native/app/src/screens/health/health-overview.tsx

```diff
@@ -550,36 +550,37 @@ export const HealthOverviewScreen: NavigationFunctionComponent = ({
               }
             />
           )}
-          {isOrganDonationEnabled && (
-            <InputRow background>
-              <Input
-                label={intl.formatMessage({
-                  id: isOrganDonorWithLimitations
-                    ? 'health.organDonation.isDonorWithLimitations'
-                    : isOrganDonor
-                    ? 'health.organDonation.isDonor'
-                    : 'health.organDonation.isNotDonor',
-                })}
-                value={`${intl.formatMessage(
-                  {
+          {isOrganDonationEnabled &&
+            (organDonationRes.loading || organDonationRes.data) && (
+              <InputRow background>
+                <Input
+                  label={intl.formatMessage({
                     id: isOrganDonorWithLimitations
-                      ? 'health.organDonation.isDonorWithLimitationsDescription'
+                      ? 'health.organDonation.isDonorWithLimitations'
```

Comment:
Git is diffing this wrong 😅 I did only add the (organDonationRes.loading || organDonationRes.data) && at the top, nothing else changed